api/bases/cinder.openstack.org_cinders.yaml

```diff
@@ -147,6 +147,24 @@ spec:
                           x-kubernetes-int-or-string: true
                         type: object
                     type: object
+                  tls:
+                    properties:
+                      api:
+                        properties:
+                          internal:
+                            properties:
+                              secretName:
+                                type: string
+                            type: object
+                          public:
+                            properties:
+                              secretName:
+                                type: string
+                            type: object
+                        type: object
+                      caBundleSecretName:
```

Comment:
I am not thrilled with the structure. I think the CA bundle should be a top level  Cinder CRD element as it affects all its services.

Because we are adding it only as a CinderAPI element, then it gets weird when things like the cronjob or dbsync need to access the CinderAPI in the spec to get the CA bundle.

I will not hold the patch because of this, as I understand this is how it has been done everywhere and now is not the time to disagree.